utils/ultralytics/train.py

```diff
@@ -43,15 +52,38 @@ def ultralytics_train_yolov5(
                 kili_api_key=api_key,
                 project_id=project_id,
                 label_types=label_types,
-                max_assets=max_assets
+                max_assets=max_assets,
             )
         )
 
-    args_from_json = " ".join(f" --{k} {v}" for k, v in json_args.items())
-    kili_print("Starting Ultralytics\' YoloV5 ...")
-    os.system(f'cd {yolov5_path} && python train.py --data kili.yaml --project "{output_path}" {args_from_json}')
+    args_from_json = reduce(
+        lambda x, y: x + y, ([f"--{k}", f"{v}"] for k, v in json_args.items())
+    )
+    print(args_from_json)
+    kili_print("Starting Ultralytics' YoloV5 ...")
+    try:
+        args = [
+            "cd",
+            f'"{yolov5_path}"',
+            "&&",
+            "python",
+            "train.py",
+            "--data",
+            "kili.yaml",
+            "--project",
+            f'"{output_path}"',
+            *args_from_json,
+        ]
+        print(args)
+        subprocess.run(
+            args,
+            check=True,
+        )
+    except subprocess.CalledProcessError:
+        kili_print("YoloV5 training crashed")
```

Comment:
Not now, but later on we could implement a KiliError class and raise the error with the stack

Reply:
why not. In fact there is a mistake there: I forgot to raise the exception after the print. Doing it.